wallet/src/main/java/com/android/identity_credential/wallet/util/CommonExtensions.kt

```diff
@@ -3,9 +3,25 @@ package com.android.identity_credential.wallet.util
 import android.content.Context
 import android.content.ContextWrapper
 import androidx.activity.ComponentActivity
+import com.android.identity.credentialtype.CredentialTypeRepository
 
+/**
+ * Attempt to get a ComponentActivity or Activity from a context
+ */
 fun Context.getActivity(): ComponentActivity? = when (this) {
     is ComponentActivity -> this
     is ContextWrapper -> baseContext.getActivity()
     else -> null
-}
\ No newline at end of file
+}
+
+/**
+ * Extension function for extracting the display name for an element name in a CredentialRequest.DataElement
+ */
+fun CredentialTypeRepository.stringValueFor(
+    docType: String,
+    namespace: String,
+    elementName: String
+) = getMdocCredentialType(docType)?.namespaces
+    ?.get(namespace)?.dataElements?.get(elementName)
+    ?.attribute?.displayName
+    ?: elementName
```

Comment:
I don't think it's overkill to make this an extension function (and I would recommend creating one if I saw code that passed an object to a function only so it can extract data from that object) but I do agree this function can live inside `ConsentPrompt`.

The paradigm of using an extension function isn't just to provide 2 or more callers an "easy time" performing some functionality, but rather, it is extending the functionality of an already existing class without modifying it. Since we are extracting data from `CredentialTypeRepository` it makes perfect sense that we extend the functionality of `CredentialTypeRepository` to easily extract data from it even if there's just 1 caller.

The alternative would be to create a normal function that accepts the repository instance parameter only so it can be used to extract its data. This style of coding isn't very pleasing or very Kotlin-esque.

---- 

In regards to the placement of this extension function, initially I placed it in a common area so it can be used by anyone in the future that uses `CredentialTypeRepository`, but I also agree that it makes sense to put it inside ConsentPrompt. (it can just as easily be moved to a common area if it's ever needed outside of ConsentPrompt)